server.js

```diff
@@ -20,13 +18,223 @@ mongoose.Promise = Promise;
 const port = process.env.PORT || 8080;
 const app = express();
 
+const Player = mongoose.model('Player', {
+  name: String,
+  year_born: Number,
+  age: Number,
+  position: String,
+  nationality: String,
+  shirt_number: Number,
+  done_goal: String
+})
+
+if (process.env.RESET_DB === 'true') {
+  const seedDatabase = async () => {
+    await Player.deleteMany({})
+
+    data.forEach((item) => {
+      const newPlayer = new Player(item)
+      newPlayer.save()
+    })
+  }
+  seedDatabase()
+}
+
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
+// ERROR HANDLING:
+app.use((req, res, next) => {
+  if (mongoose.connection.readyState === 1 ) { // 1 då restrande inte är connected. 1 = connected
+    next() // Hanterar funktionen nedan, alltså när allt fungerar som vanligt
+  } else {
+    res.status(503).json({error: 'Service unavilable'})
+  }
+})
+
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send(
+    {"Welcome":"This is an open API with Hammarbys football players 2022.",
+    "Database": "MongoDB",
+    "Endpoints": "/endpoints",
+    "Routes":[{
+    "/hammarby":"Get all the players",
+    "/hammarby/names/:name":"Get a player by name",
+    "/hammarby/ages/:age":"Get the player with a specific age",
+    "/hammarby/positions/:position":"Get player based on their position (eg: striker, midfielder, defender, goalkeeper)",
+    "/hammarby/born/:year":"Get player based on the year they were born (eg: 1991)",
+    "/hammarby/nationalitys/:nationality":"Get player based on their nationality (eg: Sweden, Denmark, Albania, Gambia)",
+    "/hammarby/goals/:goal":"Shows the players who has done at least one goal for Hammarby (yes / no)",
+    "/hammarby/numbers/:number":"Shows the player with a specific shirt number (eg: 15)"
+  }],
+  "Querys":[{
+    "/hammarby/players?shirt_number=number":"Shows the player with a specific shirt number (eg: 15)",
```

Comment:
maybe /hammarby?shirt_number=number (without players) would be a better choice because /hammarby and /hammarby/players retrieves same data??!? Or I guess you just want to try param and query to filter data for learning purpose? :)